### PR TITLE
Add AssetMantle

### PR DESCRIPTION
**Pull request by tombeynon**
Genesis and peers are from: https://github.com/AssetMantle/genesisTransactions/tree/main/mantle-1. Not sure if we want to list all peers?

The logo image is a Github avatar right now, but I can inline it in this repo if that's preferred. 

Rushed together for restake.app but happy to make any changes needed.

### CODE REVIEW

File: assetmantle/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "denom": "umntl",
+            "exponent": 0
+          },
+          {
+            "denom": "mntl",
+            "exponent": 6
+          }
+        ],
+        "base": "umntl",
+        "name": "Mantle",
```
Comment (Galadrin) [inline assetmantle/assetlist.json:18]:
```suggestion
        "name": "AssetMantle",
```

**Review by Galadrin — CHANGES_REQUESTED** [review]

File: assetmantle/chain.json [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+            {
+                "id": "13e82c193914d0db037566ff0e59dfd38ccf5193",
+                "address": "35.154.66.182:26656"
+            }
+        ]
+    },
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://rpc-assetmantle.ecostake.com",
```
Comment (Galadrin) [inline assetmantle/chain.json:316]:
```
{
    "address": "https://rpc.assetmantle.one/",
    "provider": "AssetMantle"
}
```

File: assetmantle/chain.json [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+                "provider": "ecostake"
+            }
+        ],
+        "rest": [
+            {
+                "address": "https://rest-assetmantle.ecostake.com",
```
Comment (Galadrin) [inline assetmantle/chain.json:322]:
```
{
    "address": "https://rest.assetmantle.one/",
    "provider": "AssetMantle"
}
```

File: assetmantle/chain.json [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+                "provider": "ecostake"
+            }
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "ping.pub",
+            "url": "https://explorer.postcapitalist.io/AssetMantle",
+            "tx_page": "https://explorer.postcapitalist.io/AssetMantle/tx/${txHash}"
```
Comment (Galadrin) [inline assetmantle/chain.json:331]:
```
        {
            "kind": "mintscan",
            "url": "https://www.mintscan.io/asset-mantle",
            "tx_page": "https://www.mintscan.io/asset-mantle/txs/${txHash}"
        },
        {
            "kind": "other",
            "url": "https://explorer.assetmantle.one",
            "tx_page": "https://explorer.assetmantle.one/transactions/${txHash}"
        }
            
```

**Review by Sdpierret — APPROVED** [review]

**Review by Galadrin — APPROVED** [review]

File: assetmantle/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            "denom": "mntl",
+            "exponent": 6
+          }
+        ],
+        "base": "umntl",
+        "name": "AssetMantle",
+        "display": "mntl",
+        "symbol": "MNTL",
+        "logo_URIs": {
+          "png": "https://avatars.githubusercontent.com/u/97742402?s=200&v=4"
```
Comment (sunnya97) [inline assetmantle/assetlist.json:22]: Could we pull this into this repo.  (make a subfolder called images inside the assetmantle folder)

Comment (sunnya97) [reply assetmantle/assetlist.json:22]: Going to merge for now, please do this in a new PR

**Pull request merged**